app/assets/stylesheets/pages/home.scss

```diff
@@ -1,22 +1,61 @@
 $laptop-width: 969px;
 $small-screen: em(420);
 
-#home-banner {
+#home-image {
   background: image-url("home/banner-bg.jpg");
   background-size: cover;
+  height: 660px;
+
+  @include media(max-width $medium-screen) {
+    width: 100%;
+    height: 550px;
+  }
+}
+
+#home-padding {
+  padding-top: 190px;
+
+  @include media(max-width $medium-screen) {
+    padding-top: 90px;
+  }
+}
+
+#home-screen {
+  width: 552px;
```

Comment:
Properties should be ordered background, background-size, border, border-radius, height, width